### DISCUSSION TITLE
disable/enable notifications from app

### DISCUSSION THREAD

**Discussion by a43mrk** [firebase/flutterfire#5621] [Feature request]
Hi there!

it was good if we can switch the notifications from flutter app!
I didn't find anything about this subject anywhere.

regards!

**Comment by Ehesp** [comment]
Could you provided some more details? If you mean control the users notification settings from the app, you can't do that.

**Comment by a43mrk** [comment]
Thank you @Ehesp 
yes, that feature is from users notification settings...
is there a plan to support this feature in the future?
